Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -2725,35 +2725,48 @@ private Bitmap constructBitmap(final int markerSize, final String title) {
         return Bitmap.createScaledBitmap(b, markerWidth, markerHeight, false);
     }
 
-    private Bitmap getMarkerBitmapFromView(String locationName, String imageName) {
-
+    private Bitmap getMarkerBitmapFromView(String locationName, String imageName, String specialLocationTagIcon) {
         View customMarkerView = ((LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate( imageName.equals("ny_ic_zone_pickup_marker") ? R.layout.zone_label_layout :  R.layout.marker_label_layout, null);
+        View ImageAndTextView = customMarkerView.findViewById(R.id.zone_image_and_text);
```

Comment:
Is this `zone_image_and_text` present in "R.layout.zone_label_layout"? As it will break if its not there